v3.0/faq/tidb.md

```diff
@@ -599,6 +599,12 @@ Now, there are still a few reasons for this error reporting (the latter two are
 > + For each DDL operation, the number of `schema` version changes is the same with the number of corresponding `schema state` version changes.
 > + Different DDL operations cause different number of `schema` version changes. For example, the `CREATE TABLE` statement causes one `schema` version change while the `ADD COLUMN` statement causes four.
 
+#### Error is reported when executing DDL statements under high concurrency?
+
+When executing DDL statements (such as building tables in batches) under high concurrency, a very few of them may fail due to key conflicts in the concurrent execution.
```

Comment:
```suggestion
When you execute DDL statements (such as building tables in batches) under high concurrency, a very few of these statements might fail because of key conflicts during the concurrent execution.
```